### PR TITLE
Add handler

### PR DESCRIPTION
**Pull request by qinqon**
Add nmstate-handler as a proxy to nodes
    
    It's a privileged DaemonSet that will have nmstate library installed
    with this in place nmstate-manager can exec into nmstate-handler pods
    and run nmstatectl to retrieve and set network config.
    
    Added make target "make cluster-sync-handler" to just sync the handler

### CODE REVIEW

File: deploy/handler.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    spec:
+      containers:
+        - name: nmstate-handler
+          image: REPLACE_IMAGE
+          imagePullPolicy: Always
+          volumeMounts:
+          - name: dbus-socket
+            mountPath: /run/dbus/system_bus_socket
+          env:
+          - name: NODE_NAME
```
Comment (phoracek) [inline deploy/handler.yaml]: Do we need any of those env variables?

File: build/handler/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM centos:7
+RUN yum -y install epel-release && \
+    yum -y update && \
+    yum -y install \
+        https://kojipkgs.fedoraproject.org//packages/nmstate/0.0.5/1.el7/noarch/python2-libnmstate-0.0.5-1.el7.noarch.rpm \
```
Comment (phoracek) [inline build/handler/Dockerfile]: // should probably be /

File: deploy/handler.yaml (hunk 1 above)
Comment (qinqon) [reply deploy/handler.yaml]: Nope, removed.

File: build/handler/Dockerfile (hunk 2 above)
Comment (qinqon) [reply build/handler/Dockerfile]: Done, also upgraded nmstate to 0.0.6

Was entertaining the idea of using ubi7 or even ubi8 but looks like we need subscription-manager to install deps.

File: deploy/handler.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+apiVersion: extensions/v1beta1
```
Comment (phoracek) [inline deploy/handler.yaml]: please try running this with openshift. i think we will need to add scc there

Comment (qinqon) [reply deploy/handler.yaml]: So check-patch.os-3.11.0.el7.x86_64 is passing so we are all good ?

Comment (phoracek) [reply deploy/handler.yaml]: Not sure, it may get created, but failt to schelute pods. Please trz it locally

Comment (qinqon) [reply deploy/handler.yaml]: confirmed that it fails 
        Warning   FailedCreate                      daemonset-controller                 Error creating: pods "nmstate-handler-" is forbidden: unable to validate against any security context constraint: [spec.volumes[0]: Invalid value: "hostPath": hostPath volumes are not allowed to be used spec.containers[0].securityContext.privileged: Invalid value: true: Privileged containers are not allowed]
1s        33s       3         ansible-service-broker.15a725c5e9548158            ClusterServiceBroker

Comment (SchSeba) [reply deploy/handler.yaml]: we need to add some SCC config

Comment (qinqon) [reply deploy/handler.yaml]: Done

**Pull request merged**